packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -29,18 +30,18 @@ const testContainerConfig: ITestContainerConfig = {
 
 class MockDetachedBlobStorage implements IDetachedBlobStorage {
     public readonly blobs = new Map<number, ArrayBufferLike>();
-    private blobCount = 0;
```

Comment:
Pre-existing: I think you'll find it simplifies things to convert the 'id' to a string in createBlob() and keep a map of <string, ArrayBufferLike>.

Reply:
(I like removing the redundant 'blobCount'!)